rootfs/api/urls.py

```diff
@@ -45,7 +45,8 @@
     url(r"^apps/(?P<id>{})/containers/?".format(settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'get': 'list'})),
     # application domains
-    url(r"^apps/(?P<id>{})/domains/(?P<domain>[-\._\w]+)/?".format(settings.APP_URL_REGEX),
+    # FIXME make sure * is only matched at the start
+    url(r"^apps/(?P<id>{})/domains/(?P<domain>[-\*\._\w]+)/?".format(settings.APP_URL_REGEX),
```

Comment:
I think bounding the overall regex with slashes ensures that case won't match. I linked to my test expressions below.

    /(?P<domain>\**\.?[-\._\w]+)/

![Regular expression visualization](https://www.debuggex.com/i/b-lamaeKEw0ZrsV8.png)

[Debuggex Demo](https://www.debuggex.com/r/b-lamaeKEw0ZrsV8)